### DISCUSSION TITLE
Make select next/prev item jump between the first item and the last item in one move.

### DISCUSSION THREAD

**Discussion by JoeMob** [hrsh7th/nvim-cmp#1582] [Q&A]
I'm trying to make cmp work like vscode, in vscode the compteletion doesn't have an empty item before the first item or after the last item.
I come up with two ways to approch it, one is making the keymap for select next/prev item jumps twice when it's the first or last item, but I didn't find the api to get the selected item's serial number in wiki. The other is removing the no action item from the completion list, but I didn't find how to configure cmp to remove the no action item either.

**Comment by Shougo** [comment]
Hm.  But current behavior is same with native completion.
VSCode is not Vim.
So I think native completion compatibility is important.

**Reply by JoeMob** [reply]
I don't want to change the current behavior of nvim-cmp, I just want to configure it to work as I want. Maybe my title is not precise.
